default/default.lass

```diff
@@ -67,7 +67,13 @@
    :display inline
    :margin 0 :padding 0
    (li :display inline
-       :padding 0 0.2em))))
+       :padding 0 0.2em))
+  ("#author-label"
+   :vertical-align top)
+  ("#authors"
+   :display inline-block
+   :margin 0 :padding 0
+   :list-style none)))
```

Comment:
Instead of this, it should be under `"#system"`:
```
("#author"
  (label :vertical-align top)
  (ul :display inline-block
      :margin 0 :padding 0
      :list-style none))
```